packages/block-library/src/site-logo/edit.js

```diff
@@ -162,7 +177,7 @@ const SiteLogo = ( {
 	}
 
 	if ( ! isResizable || ! imageWidthWithinContainer ) {
-		return <div style={ { width, height } }>{ imgWrapper }</div>;
+		return <div style={ { width, height } }>{ img }</div>;
```

Comment:
```suggestion
		return (
			<div style={ { width, height } }>
				<ImageWrapper
					isLink={ isLink }
					href={ siteUrl }
					title={ title }
					classes={ classes }
				>
					{ img }
				</ImageWrapper>
			</div>
		);
```

In the original code, the `img` element should have been wrapped in an `a` element if `isLink` was `true`.